src/store.js

```diff
@@ -25,9 +25,11 @@ export class Store {
       walletUnlocked: false, // Is the wallet unlocked
       lndReady: false, // Is lnd process running
       syncedToChain: false, // Is lnd synced to blockchain
+      isSyncing: false, // Is lnd syncing to chain or filter headers
```

Comment:
`syncedToChain` property 1 line above is only being used in `initLoaderSyncing` action. Should we use `isSyncing` there as well?

In addition the naming of these 2 is confusing. I do understand what the difference is by reading the comment, but it won't be apparent in other places in the code.